File: WEB/MyXSpace.Web/ClientApp/src/app/models/tenant.model.ts

export class Tenant
{
  public id: string;// GUID
  public name: string; //= BrandName
  public host: string;
  public theme: string;
  public connectionString: string;
  public isActive: boolean;

  constructor(name?: string)
  {
      this.name = name;
  }

}

File: WEB/MyXSpace.Web/DTO/TenantModel.cs

namespace MyXSpace.WebSPA.DTO
{
    public class TenantModel
    {
        /// <summary>
        ///  Name of the customer brand : MyAdsearch, MySigmar, ...
        /// </summary>
        public string Name { get; set; }

        public string Host { get; set; }

        private string Theme { get; set; }

        /// <summary>
        /// DB Connection string for specific tenant
        /// </summary>
        public string ConnectionString { get; set; }

        public bool IsActive { get; set; } = true;
    }
}